src/components/Rockets.js

```diff
@@ -1,6 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import getRockets from '../redux/actions/rockets';
 
 function Rockets() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getRockets());
+  }, []);
+
```

Comment:
Every time I go from one route to rockets a fetch is made.
You can only do the fetch once. So when you leave the rockets page and return again, it should not make the fetch again.

![image](https://user-images.githubusercontent.com/72808808/141204899-1fae72c2-4a09-4f2c-b311-040a9c953060.png)